cli/internal/publish/images/images.go

```diff
@@ -51,10 +72,20 @@ func ProcessDocument(ctx context.Context, c *cloudquery_api.ClientWithResponses,
 	fmt.Println("Preparing to upload images...")
 
 	reqs := make([]cloudquery_api.TeamImageCreate, 0, len(ims))
-	for k := range ims {
+	for k, v := range ims {
+		if len(v) == 0 {
+			continue
+		}
+		absFile := v[0].absFile
```

Comment:
Not sure why this is an array, looks like we're using only the first file below anyways

Reply:
Looks like `findMarkdownImages` returns each image reference separately (and they would all have the same absFile value)